status-reports/2019-01-26.md

```diff
@@ -0,0 +1,17 @@
+Release 1.16
+
+Scope Status
+* Basic servlet support [85%]
+* Description for Takes based REST services [none]
+* Replace Guava with Cactoos [60%] (30 class dependencies left)
+* Null removal [50%] (44 matches left)
+
+Issues:
+* Takes does not comply fully to EO standards anymore
+* Project needs funding.
+* Project is developing at a slow pace because albeit having a lot of 
+developers most of them are not working for free, so they did not receive tasks 
+
+Risks
+* Cactoos may not provide all features to fully replace Guava [1/7]
```

Comment:
@paulodamaso `cactoos-http` overleap `takes` funcionalities isn't a risk too?

Reply:
@fabriciofx I don't think so; it would be a risk if we decided to use, since it could not provide all funcionality we need.

Reply:
@paulodamaso Right, but in the future they can overleap their functionalities... it shouldn't be considered in any time?

Reply:
@fabriciofx Yes, in the future we will have this